### PR TITLE
Add support for IOST addresses

### LINKED ISSUE #136: Add support for IOST addresses in encoding library
Here is the encoding library: https://github.com/ensdomains/address-encoder

Example: https://github.com/ensdomains/address-encoder/pull/47/files

Discord for dev help: https://discord.com/invite/AskZbFx

Please provide a reference link so we can check you did the encoding correctly.

We'd like to keep the encoding library as small as possible, so please be efficient in your use of space.

We will likely be reviewing PRs and paying out bounties more or less on a weekly basis. No need to ping us to check your PR, we will definitely get to it.

Thanks for your help!

### PR DESCRIPTION
**Pull request by Echolon166**
## Issue number
Closes #136 

## Description

Add support for IOST addresses

## References
- https://developers.iost.io/docs/en/2-intro-of-iost/Account.html
- https://developers.iost.io/docs/en/7-iost-js/KeyPair-class.html
- https://github.com/iost-official/iost.js/blob/84094c3ec2427031855d2cbf071fbfaeb81256b5/lib/crypto/key_pair.js#L9
- https://github.com/iost-official/iost.js/blob/84094c3ec2427031855d2cbf071fbfaeb81256b5/lib/crypto/key_pair.js#L65

## Checklist:

- [x] My code follows the code style of this project.
- [x] My code implements all the required features.

### CODE REVIEW

**Comment by makoto** [comment]
@Echolon166  Please resolve conflicts

**Comment by makoto** [comment]
@Echolon166  Please resolve conflicts

**Comment by Echolon166** [comment]
@makoto Resolved.

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ This library currently supports the following cryptocurrencies and address forma
  - HIVE (base58+ripemd160-checksum)
  - HNS
  - ICX
+ - IOST (base58)
```
Comment (makoto) [inline README.md:57]: Base change this to base58, no-check

Comment (Echolon166) [reply README.md]: Done.

**Pull request merged**